### PR TITLE
Added tests for cashier-iab module

### LINKED ISSUE #5: Write tests for all classes

### PR DESCRIPTION
**Pull request by nbarishok**
Closes #5 

Added tests to cover functionality of the next classes:`InAppBillingProduct`, `InAppBillingV3Vendor`

Extracted test data that is used across different classes into `InAppBillingTestData` class

### CODE REVIEW

File: cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java [hunk 1]
```diff
@@ -189,6 +189,8 @@ public boolean available() {
     return available && api.available() && canPurchaseAnything();
   }
 
+  int getRequestCode() {  return requestCode; }
```
Comment (philippb) [inline cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java]: Please make this in a new line and follow code style from the rest of the app

File: cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java [hunk 2]
```diff
@@ -288,7 +290,7 @@ public void consume(Context context, Purchase purchase, ConsumeListener listener
   }
 
   @Override
-  public void getInventory(Context context, Collection<String> itemSkus, Collection<String> subSkus,
+  public void getInventory(Context context, Collection<String> inappSkus, Collection<String> subSkus,
```
Comment (philippb) [inline cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java]: What is the reason for this change from `itemSkus` to `inappSkus` across this file?

File: cashier-iab/src/test/java/com/getkeepsafe/cashier/iab/InAppBillingTestData.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            "S4OFDDbm7aK8SlGPEQJBAN+YlMb4PS54aBpWgeAP8fzgtOL0Q157bmoQyCokiWv3\n" +
+            "OGa89LraifCtlsqmmAxyFbPzO2cFHYvzzEeU86XZVFkCQQDQRWQ0QJKJsfqxEeYG\n" +
+            "rq9e3TkY8uQeHz8BmgxRcYC0v43bl9ggAJAzh9h9o0X9da1YzkoQ0/cWUp5NK95F\n" +
+            "93WTAkEAxqm1/rcO/RwEOuqDyIXCVxF8Bm5K8UawCtNQVYlTBDeKyFW5B9AmYU6K\n" +
+            "vRGZ5Oz0dYd2TwlPgEqkRTGF7eSUOQJAfyK85oC8cz2oMMsiRdYAy8Hzht1Oj2y3\n" +
+            "g3zMJDNLRArix7fLgM2XOT2l1BwFL5HUPa+/2sHpxUCtzaIHz2Id7QJATyF+fzUR\n" +
+            "eVw04ogIsOIdG0ECrN5/3g9pQnAjxcReQ/4KVCpIE8lQFYjAzQYUkK9VOjX9LYp9\n" +
+            "DGEnpooCco1ZjA==";
+
+    String TEST_PUBLIC_KEY_1 = "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQC16JSHANSyHGA5ztp32pW5Dg9l\n" +
```
Comment (philippb) [inline cashier-iab/src/test/java/com/getkeepsafe/cashier/iab/InAppBillingTestData.java]: you user `TEST_PUBLIC_KEY_1` for test that indicate a valid key or transaction and `TEST_PUBLIC_KEY_2` for tests that aim for catching an invalid key as I understand.

Should the variable names not indicate that? Looks like `TEST_PUBLIC_KEY_1` matched with `TEST_PRIVATE_KEY` while `TEST_PUBLIC_KEY_2` doesn't.

**Comment by philippb** [comment]
@xiphirx if you feel compelled, feel free to have a look at this PR. @nbarishok has been adding tests around your old Cashier project.

**Comment by philippb** [comment]
@nbarishok Can you please add a more detailed description for both Pos you opened, so we can read what happened here also in the future. 
Thank you

File: cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java (hunk 2 above)
Comment (nbarishok) [reply cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java]: Just noticed that 
`
  public static final String PRODUCT_TYPE_ITEM = "inapp";
  public static final String PRODUCT_TYPE_SUBSCRIPTION = "subs";
`
So `itemSkus -> inappSkus` felt reasonable. Can revert back, or `PRODUCT_TYPE_ITEM -> PRODUCT_TYPE_INAPP` for consistency

File: cashier-iab/src/test/java/com/getkeepsafe/cashier/iab/InAppBillingTestData.java (hunk 3 above)
Comment (nbarishok) [reply cashier-iab/src/test/java/com/getkeepsafe/cashier/iab/InAppBillingTestData.java]: yes, you are right, I'll rename them

File: cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java (hunk 1 above)
Comment (nbarishok) [reply cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java]: yes, surprise even for me that I've written it like that ))
